test/jdk/javax/swing/JRadioButton/8075609/bug8075609.java

```diff
@@ -51,9 +51,8 @@ public static void main(String[] args) throws Throwable {
             SwingUtilities.invokeAndWait(bug8075609::createAndShowGUI);
 
             robot = new Robot();
-            Thread.sleep(100);
             robot.waitForIdle();
-
+            robot.delay(1000);
```

Comment:
Test is passing in my local machine without this extra delay.

Does the test pass on CI machines without this increased delay or the delay is required to make the test more stable ?

Reply:
Its not needed strictly par se and it passed without it also but I modified as the test is using Robot class so better to use robot method..

Reply:
Ok..